common-utils/src/common/deployment/__init__.py

```diff
@@ -213,40 +202,22 @@ def push_image(self, account_id: str) -> str:
 class ProjectEnvs(BaseModel):
     """Model that represents the project environment.
     This will store the list of docker envs to build and push.
+    The pyproject.toml configuration is used here.
     """
 
     docker_envs: list[FlowDockerEnv]
+    pyproject_metadata: PyProjectMetadata
 
     def _push_environments(self, build_only: bool = False) -> None:
         """Build and optionally push ECR Image.
 
         Args:
             build_only (bool, optional): _description_. Defaults to False.
         """
-        account_id = get_aws_account_id()
         for env in self.docker_envs:
             env.build_image()
             if not build_only:
-                env.push_image(account_id)
-
-    def _create_file_systems(self, build_only: bool = False) -> None:
-        """Create Prefect S3 filesystem for project.
-
-        Args:
-            build_only (bool, optional): Switch to only run if True. Defaults to False.
-        """
-        if not build_only:
-            project_name = PYPROJECT_METADATA.project_name
-            for deployment_type in ["test", "live"]:
-                bucket_name = (
-                    f"data-flows-prefect-fs-{ENVIRONMENT_TYPE}-{deployment_type}"
-                )
-                block_name = f"{project_name}-{ENVIRONMENT_TYPE}-{deployment_type}"
-                flows_fs = S3(bucket_path=f"{bucket_name}/{project_name}")
-                result = flows_fs.save(block_name, overwrite=True)
-                LOGGER.info(
-                    f"Filesystem block: {block_name} created with id: {result}..."
-                )
```

Comment:
removing s3 flow storage in favor of github storage.  Our repo is public, so this just makes sense.  This allows us to better map deployments to branches.